Tests/lib/taskRunner.ts

```diff
@@ -1,15 +1,14 @@
 /// <reference path="../../definitions/node.d.ts"/>
 /// <reference path="../../definitions/Q.d.ts"/>
 /// <reference path="../../definitions/shelljs.d.ts"/>
-/// <reference path="../../definitions/vsts-task-lib.d.ts"/>
 
 import Q = require('q');
 import events = require('events');
 import fs = require('fs');
 import path = require('path');
 import child_process = require('child_process');
 import shell = require('shelljs');
-import tcm = require('vsts-task-lib/taskcommand');
+import tcm = require('./vsts-task-lib/taskcommand');
```

Comment:
This was extremely confusing. vsts-task-lib was added to the package.json, and so the real vsts-task-lib was getting imported here. In the gulp scripts, the mock node_module was actually installed to _build/Tests/lib/node_modules/vsts-task-lib.

The only reason this is imported is because commandFromString from the lib is used. Other than that, the runner doesn't need it at all.


The reason this manifested in a problem for the Gradle task, is because commandFromString in the mock lib is different from